src/data_source/storage/sql.rs

```diff
@@ -1781,4 +1953,105 @@ mod test {
         assert_eq!(cfg.host, "localhost");
         assert_eq!(cfg.port, 5432);
     }
+
+    #[async_std::test]
+    async fn test_target_period_pruning() {
+        setup_test();
+
+        let db = TmpDb::init().await;
+        let port = db.port();
+        let host = &db.host();
+
+        let migrations = include_migrations!("$CARGO_MANIFEST_DIR/migrations").collect::<Vec<_>>();
```

Comment:
You shouldn't need `include_migrations!` here, whatever is in the `migrations` directory is executed by default